File: src/main/java/club/deneb/client/features/modules/player/Fastplace.java
package club.deneb.client.features.modules.player;


import club.deneb.client.features.Category;
import club.deneb.client.features.Module;

/**
 * @author 086
 */
@Module.Info(name = "Fastplace", category = Category.PLAYER, description = "Nullifies block place delay")
public class Fastplace extends Module {

    @Override
    public void onTick() {
        mc.rightClickDelayTimer = 0;
    }
}

File: src/main/java/club/deneb/client/features/modules/player/FastPlace.kt
package club.deneb.client.features.modules.player

import club.deneb.client.features.Category
import club.deneb.client.features.Module

@Module.Info(name = "FastPlace", category = Category.PLAYER, description = "Nullifies block place delay")
class FastPlace : Module() {
    override fun onTick() {
        mc.rightClickDelayTimer = 0
    }
}